include/deal.II/base/config.h.in

```diff
@@ -38,6 +38,7 @@
 #cmakedefine DEAL_II_WITH_ADOLC
 #cmakedefine DEAL_II_WITH_ARPACK
 #cmakedefine DEAL_II_WITH_ASSIMP
+#cmakedefine DEAL_II_WITH_TET_MESH_SUPPORT
```

Comment:
my OCD demands it: Would you mind sorting alphabetically? :-)

Reply:
Done!